packages/augur-sdk/src/utils.ts

```diff
@@ -120,7 +121,12 @@ export function calculatePayoutNumeratorsValue(
   if (!payout) return null;
   if (payout.length === 0) return null;
 
+
   if (isScalar) {
+    if (isInvalidScalar(payout)) {
+      return MALFORMED_OUTCOME;
```

Comment:
nope, you are mixing two different concepts.

Reply:
:(